documentation/getstarted/advanced/migrationtesting.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: documentation
+menu: migrationtesting
+subtitle: 'Tutorial: Testing Flyway migrations in a CI pipeline'
+---
+# Tutorial: Testing Flyway migrations in a CI pipeline
+
+This tutorial will guide you through the process of setting up Flyway migration tests in a CI pipeline using [Spawn](spawn.cc) to provision a database on the fly.
+
+## Prerequisites
+
+Install Spawn by visiting the [getting started documentation](https://www.spawn.cc/docs/getting-started.html) and following the installation steps.
+
+This tutorial will assume you have a database with Flyway migrations already set up, and migration scripts stored in your repository.
```

Comment:
This reads like I'm going to need one of my real databases to follow along with. Is the intention for the user to follow this tutorial with their real database, or to use Spawn to take a run at doing it with something that's disposable? It feels like framing the tutorial around the latter (but making it clear that it's the same steps for something real) might be the better lower-stakes option?

Reply:
That was our intention, people to follow along with their real database so they can implement some migration testing before they run flyway migrations against their real database.

Following along with their prod database isn't really high-stakes - they're never touching their production data, the point that we use Spawn is taking a backup of it - but I see your point, I guess we could use some dummy data in Spawn as their "production database" but we didn't think that'd add much value? We'd expect (hope!) people to come along to this article to solve real-world problems they're hitting, and find it easy to add a testing stage in their CI pipeline - that's ultimately what they can take away from this.